ci/common/common.sh

```diff
@@ -7,7 +7,7 @@
 # ${3}: Line number.
 require_environment_variable() {
   local variable_name="${1}"
-  eval "local variable_content=\"\${${variable_name}:-}\""
+  local variable_content="${variable_name:-}"
```

Comment:
The code is entirely different. Before `$variable_content` was set to variable content, not surprisingly. Now you set it to variable name.